### ISSUE TITLE
Running in parallel with paratest

### ISSUE THREAD

**Issue by g4b0** [ergebnis/phpunit-slow-test-detector#624]
#### Steps required to reproduce the problem

1. install paratest (https://github.com/paratestphp/paratest)
2. configure phpunit.xml for using SlowTestDetector
3. run `vendor/bin/paratest`

#### Expected Result
Slow tests listes as when running `vendor/bin/phpunit`

#### Actual Result
Tests are executed (in parallel), but no output about slow tests

**Comment by localheinz** [comment]
Duplicate of #280.
